src/features/account/MoreOptionMenu.tsx

```diff
@@ -0,0 +1,27 @@
+import { t } from '@lingui/macro';
+import type { FunctionComponent } from 'react';
+
+import { IconMenu } from '../../components/IconMenu';
+import {
+  ExportOutlineIcon,
+  MoreOptionIcon,
+  ShareIcon,
+  UserCircleAddIcon,
+  WarningFilledIcon,
+} from '../../components/icons';
+import { MenuItemCard } from '../../components/MenuItemCard';
```

Comment:
Some of these imports can be simplified if you export the components from `index.ts`.

Reply:
has add `IconMenu` and `MenuItemCard` into `index.ts` under component folder, and simplify above imports.